src/tabs/test/tabs.spec.js

```diff
@@ -98,6 +98,10 @@ describe('tabs', function() {
       titles().eq(1).find('> div').click();
       expect(scope.deselectFirst).toHaveBeenCalled();
     });
+
+    it('should expose controller as "tab" scope variable', function() {
+      expect(typeof elm.find('.uib-tab').first().isolateScope().tab).toEqual('object');
```

Comment:
Using a typeof check is not great, and this test is a little too vague. It would be better to test against what it equals exactly.